CIP-0068/README.md

```diff
@@ -191,7 +191,7 @@ metadata =
 ; A valid Uniform Resource Identifier (URI) as a UTF-8 encoded bytestring.
 ; The URI scheme must be one of `https` (HTTP), `ipfs` (IPFS), `ar` (Arweave) or `data` (on-chain).
 ; Data URLs (on-chain data) must comply to RFC2397.
-uri = bounded_bytes ; UTF-8
+uri = bounded_bytes / [ * bounded_bytes ] ; UTF-8
```

Comment:
URI is also defined on Line 310 of this file, can we please sweep through the entire spec and make sure that we make this same change in all places where we define a URI?

Reply:
Also on Line 436

Reply:
Ah interesting should they be defined in 2 places?

Reply:
Not really, it should be done more globally but since we have prose mixed with markdown mixed with CDDL mixed with other examples, it's better that each different token type has its own encapsulated CDDL specification rather than relying on a disconnected text blob earlier in the document.